content/chapitres/cli.adoc

```diff
@@ -0,0 +1,209 @@
+[{invert}]
+= Guide de survie en ligne de commande
+
+Remise à niveau / Rappels
+
+[{invert}]
+== CLI
+
+- 🇬🇧 CLI == "Command Line Interface"
+- 🇫🇷 "Interface de Ligne de Commande"
+
+== REPL
```

Comment:
True that. I found your mention of it really interesting but I realize that it would be useful for 101 CLI course or at least when people "discover it" (or a "fundamentals of CLI").

But since this chapter is aimed at being a "survival guide", then better off without this. Thanks!